packages/pulpcore/python-itypes/python-itypes.spec

```diff
@@ -2,12 +2,12 @@
 %global pypi_name itypes
 
 Name:           python-%{pypi_name}
-Version:        1.1.0
-Release:        2%{?dist}
+Version:        1.2.0
+Release:        1%{?dist}
 Summary:        Simple immutable types for python
 
 License:        BSD
-URL:            http://github.com/tomchristie/itypes
+URL:            http://github.com/PavanTatikonda/itypes
```

Comment:
This repo is archived. Looks like it was a dependency of `coreapi` which is now also archived. Are the pulpcore devs aware of this?

Reply:
I pinged them on IRC, yes.

Reply:
and opened https://pulp.plan.io/issues/6583